lib/keycloak-admin/client/user_client.rb

```diff
@@ -31,6 +31,32 @@ def update(user_id, user_representation_body)
       )
     end
 
+    # Add User-ID to group
+    def add_group(user_id, group_id)
+      RestClient::Request.execute(
+        @configuration.rest_client_options.merge(
+          method: :put,
+          url: "#{users_url(user_id)}/groups/#{group_id}",
+          payload: {
```

Comment:
Are you sure the payload should not be empty? (https://www.keycloak.org/docs-api/12.0/rest-api/index.html#_users_resource)

Reply:
i have tested it with my Keycloak Version 12.0.1, and no Payload is needed to use this functionality. i could add it soon.

Reply:
I will fix it myself. Thanks for your contribution :)